lib/Twig/Extensions/Extension/Array.php

```diff
@@ -50,3 +52,5 @@ function twig_shuffle_filter($array)
 
     return $array;
 }
+
+class_alias('Twig_Extensions_Extension_Array', 'Twig\Extensions\Extension\ArrayExtension', false);
```

Comment:
@fabpot what do you think about adding the suffix for all extensions ?

@nicolas-grekas for reference, we have the suffix everywhere for extensions defined in Symfony.